lib/routing.py

```diff
@@ -96,8 +106,8 @@ def url_for(self, func, *args, **kwargs):
                 path = rule.make_path(*args, **kwargs)
                 if path is not None:
                     return self.url_for_path(path)
-        raise RoutingError("No known paths to '{0}' with args {1} and "
-                           "kwargs {2}".format(func.__name__, args, kwargs))
+        raise RoutingError("No known paths to '{0}' with args {1} "
```

Comment:
I don't see a good reason for this change.

Reply:
pylint was complaining about line length, and showing a red X on my PR

Reply:
PyLint was complaining about line length and marking my PR with a red X

Reply:
I see. I have a PR open to raise this to 160. But in that case I understand.